src/session.ts

```diff
@@ -198,6 +204,7 @@ export class Session {
         // Create response template to this transact call
         const result: TransactResult = {
             chain: this.chain,
+            keys: [],
```

Comment:
Store an array of all the public keys used to sign the request to both provide in the response and use during validation of those signatures.